React/Base/RCTBridge.m

```diff
@@ -322,29 +385,24 @@ - (NSString *)description
     const RCTExportValue mach_header = (RCTExportValue)info.dli_fbase;
     const RCTExportSection *section = RCTGetSectByNameFromHeader((void *)mach_header, "__DATA", "RCTExport");
 
-    if (section == NULL) {
-      return;
-    }
+    if (section != NULL) {
```

Comment:
No. The logic is inverted because the function can no longer just exit if the section is not found - it must process any methods found by the "methodsToExport" class method. It also fixes a subtle bug - this function should be returning a RCTSparseArray of registered methods.